app/src/org/commcare/fragments/personalId/PersonalIdBiometricConfigFragment.java

```diff
@@ -124,50 +130,56 @@ private void reportAuthSuccess() {
     }
 
     private void refreshAuthenticationOptions() {
-        BiometricsHelper.ConfigurationStatus fingerprintStatus = BiometricsHelper.checkFingerprintStatus(
-                getActivity(), biometricManager);
-        BiometricsHelper.ConfigurationStatus pinStatus = BiometricsHelper.checkPinStatus(getActivity(),
-                biometricManager);
-
-        if (fingerprintStatus == BiometricsHelper.ConfigurationStatus.NotAvailable &&
-                pinStatus == BiometricsHelper.ConfigurationStatus.NotAvailable) {
-            Logger.exception("No biometrics available", new Exception("No biometric or PIN options available"));
-            navigateForward(true);
-            return;
+        String errorMsgForMinHardwareForSecurity = BiometricsHelper.hasMinHardwareForSecurity(biometricManager,getActivity(),personalIdSessionDataViewModel.getPersonalIdSessionData().getRequiredLock());
```

Comment:
it seems a bit weird for these `has` methods to reply with error String instead of booleans and this might be a bit cleaner if we abstract out the error handing in a separate independent method.